### ISSUE TITLE
Compiler error for extern "C" struct

### ISSUE THREAD

**Issue by Martinsos** [ispc/ispc#478]
simple.cpp:
    #include "simple_ispc.h"
    extern "C" struct Test {
        int b;
    };
    int main() {
        ispc::simple();
    }

simple.ispc:
    extern "C" struct Test;
    export void simple() {
        Test t;
    }

compile command:
ispc simple.ispc -o simple_ispc.o -h simple_ispc.h

Error message:
func.cpp(423): FATAL ERROR: Function verificication failed

---

**\* Please file a bug report at https://github.com/ispc/ispc/issues
**\* (Including as much information as you can about how to reproduce this error).
**\* You have apparently encountered a bug in the compiler that we'd like to fix!

---

main.cpp(204): FATAL ERROR: Unhandled signal sent to process; terminating.

ispc version: 1.3.0
machine: Intel(R) Core(TM)2 Quad CPU    Q6600  @ 2.40GHz

**Comment by aneshlya** [comment]
Here is a repro: https://godbolt.org/z/991h19EEd

**Comment by aneshlya** [comment]
@nurmukhametov , can it be closed?

**Comment by nurmukhametov** [comment]
Yes, sure. It is fixed by https://github.com/ispc/ispc/pull/3255
